lib/net/nforceif/src/sys_arch.c

```diff
@@ -91,209 +64,176 @@ sys_init(void)
 {
 }
 
-#if 0
-u32_t
-sys_jiffies(void)
-{
-    struct timespec ts;
-
-    clock_gettime(CLOCK_MONOTONIC, &ts);
-    return ts.tv_sec * 1000000000L + ts.tv_nsec;
-}
-#endif
-
-#if !NO_SYS
-
 sys_thread_t
 sys_thread_new(const char *name, lwip_thread_fn function, void *arg, int stacksize, int prio)
 {
     struct sys_thread *st = NULL;
```

Comment:
The variable `st` is no longer used and could be deleted.

Reply:
Removed.